sjb/config/common/test_cases/origin_built_installed_release.yml

```diff
@@ -48,7 +48,9 @@ extensions:
       title: "tag openshift/origin for system containers"
       repository: "aos-cd-jobs"
       script: |-
-        sudo docker tag openshift/origin:$( cat ./ORIGIN_TAG ) openshift/origin:$( cat ./ORIGIN_TAG ).$(cat ORIGIN_PKG_VERSION | grep -Po "^\-\w\.\w\.(\w)" | cut -d '.' -f 3) || true
+        if [[ "${PULL_BASE_REF}" == "release-3.9" ]]; then
+          sudo docker tag openshift/origin:$( cat ./ORIGIN_TAG ) openshift/origin:$( cat ./ORIGIN_TAG ).$(cat ORIGIN_PKG_VERSION | grep -Po "^\-\w\.\w\.(\w)" | cut -d '.' -f 3) || true
```

Comment:
We already have a 39 fork of this job.  Don’t add this here.

Reply:
Right, updated `_release_39` instead